modules/ossm-security-mtls.adoc

```diff
@@ -52,16 +51,16 @@ Create a destination rule to configure {ProductShortName} to use mTLS when sendi
 
 [source,yaml]
 ----
-apiVersion: "networking.istio.io/v1alpha3"
-kind: "DestinationRule"
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
 metadata:
-  name: "default"
-  namespace: <CONTROL_PLANE_NAMESPACE>
+  name: default
+  namespace: <namespace>
 spec:
-  host: "*.local"
+  host: “*.<namespace>.svc.cluster.local”
   trafficPolicy:
-    tls:
-      mode: ISTIO_MUTUAL
+   tls:
+     Mode: ISTIO_MUTUAL
```

Comment:
Don't remove the space from the indentation.

`Mode` should be `mode`.

Reply:
`trafficPolicy` is 2 spaces in. `tls` is 3. `mode` is 4. Are you saying tls: is in the right place?